collect.py

```diff
@@ -15,6 +15,11 @@
     chrome_bin = "tools/chrome.app/Contents/MacOS/chrome"
     nightly_bin = 'tools/Nightly.app/Contents/MacOS/firefox'
 
+elif sys.platform.startswith("win32"):
+    path = os.getcwd()
```

Comment:
Are you sure this is needed?

Reply:
i thought it will need the full directory. okay, i will confirm it, currently, i don't have the machine now.

Reply:
yes sir, it is required. without it, compability errors are shown.

Reply:
What errors?

Reply:
https://pastebin.mozilla.org/9077598

Reply:
What is the value of nightly_bin that triggers this error?

Reply:
How to check this on windows machine ?

Reply:
and it starts functioning normally when path is added.

Reply:
'\\tools\\Nightly\\firefox.exe'

Reply:
its "\\\"

Reply:
actually its two times \

Reply:
Have you tried using a relative path? Without the initial `\` before tools

Reply:
still it is present even without initial \\

Reply:
means the error still comes.
errorr occurs when i use \\tools\\Nightly\\firefox.exe
but not in case of  os.getcwd() + \\tools\\Nightly\\firefox.exe
what is the downside of using os.getcwd?

Reply:
apologies me , its working 😃 now

Reply:
i ran  the script after rebooting the machine and its working.